script.js

```diff
@@ -97,10 +97,99 @@ class CircleMover extends Mover {
 	}
 }
 
+class Draggable {
+	constructor(x,y,w,h){
+		this.dragging = false;
+		this.rollover = false;
+		this.x = x;
+		this.y = y;
+		this.w = w;
+		this.h = h;
+		this.offsetX = 0;
+		this.offsetY = 0;
+	}
+
+	over(){
```

Comment:
More descriptive name for `over`?

Reply:
Changed to mousedOver(). Slightly more descriptive I guess.